### ISSUE TITLE
Frontend suggestions

### ISSUE THREAD

**Issue by pradumangoyal** [IMGIITRoorkee/omniport-frontend-faculty-profile#8]
1. I think do not show description in Miscellaneous section on the front card, the description can be really large sometimes so showing heading only will suffice
2. Add/Edit models in some of the section is really small for example Miscellaneous, Associate Scholars, Interest etc.. I think it is better to make all of them of the same size. Small modals are difficult to edit.

@nisarg73 @algomaster99 What do you think of these?

**Comment by nisarg73** [comment]
1. I agree with this, but then the only place where faculty can see description of a Miscellaneous object is by going into edit modal of an object. Plus I think heading alone won't be self explanatory for identifying an object. For example, in old facapp there is a heading of "Innovation in Teaching", so if there were multiple instances of this particular heading, without their description, all of them would look same. (Ofcourse, this assumes that faculty won't be setting explicit headings, like "Innovation in Teaching in MAN-206", if this is the case then there isn't any issue in not rendering description.)

2. Currently, Add/Edit modals occupy only required space (i.e. on the basis of size of input fields inside). Which makes sense actually imo. For example, Interest section's modal only have one input field, so it won't look good if we have a modal of large size for this. If current one seems to too small to edit from user's pov, one solution I can think for this is, setting a minHeight and/or minWidth to relevant input fields to increase their sizes.

**Comment by pradumangoyal** [comment]
@nisarg73 
1. Actually heading is a unique field with the faculty, So there will be only one heading named "Innovation in Teaching" and all the corresponding points will go in points in the same text box. Description can be as large as seen in the attached screenshot 
![image](https://user-images.githubusercontent.com/34851451/86319094-44bf9300-bc51-11ea-96ad-0de2ad658fe8.png) I will even suggest to exclude it in `__str__` representation of the Django model
2. If that's the case, it's better to pass a size attribute in the Model, already implemented in the Semantic. `Event` or `Paper` modal seems to have an idea size, so it is better to make all of them to that and set the fields to the fluid type. If you see the screenshot attached, it's quite difficult to edit the miscellaneous section. So, I suggest it is better to have a consistent size with the `size` prop.
![image](https://user-images.githubusercontent.com/34851451/86319254-c0214480-bc51-11ea-9538-fff604f2e9a9.png)

**Comment by pradumangoyal** [comment]
The 1st suggestion is for the front display as well as in the Priority setting modal

**Comment by nisarg73** [comment]
@pradumangoyal 
1. Oh okay, in that case I totally agree on this. So should I proceed to implement this?

2. Actually, Dimmer component is used for both Add/Edit options, and it doesn't have a size prop of its own.
As you mentioned, Event and Project seems larger in size, because they have a grouped field with 2 input fields (start date, end date) with equal width. Which increase the overall size of the segment. For demonstration, if I remove them as a grouped field and make it individual fields, Event would look like this: 
![Screenshot_2020-07-02 FacApp](https://user-images.githubusercontent.com/25002246/86322879-63c22300-bc59-11ea-90e7-c5978b7d8b29.png)

Which is same as the size of normal input field. I suggested to add a minWidth and/or minHeight prop to input fields, because unlike Modal, Dimmer component doesn't have a size prop.
